lib/perl/Genome/Model/Build/SomaticValidation.pm

```diff
@@ -319,4 +321,48 @@ sub get_feature_list_from_reference {
     return $self->reference_sequence_build->get_feature_list($feature_list_accessor);
 }
 
+sub individual {
+    my $self = shift;
+    return $self->subject;
```

Comment:
@gatoravi @jasonwalker80 Both SomVal and SomVar build can have samples or individuals as their subject so I plan on changing this to a generalized implementation to get to the subject. However, SomVal can also have a taxon as a subject. How should this case be handled?